### ISSUE TITLE
Data access request "needs credentialing"

### ISSUE THREAD

**Issue by bemoody** [MIT-LCP/physionet-build#1508]
project/views.py line 1938:
```
    needs_credentialing = proj.access_policy == AccessPolicy.CREDENTIALED and not user.is_credentialed
```
Intention here is that if access policy is credentialed then the person *cannot* submit a data access request until they are credentialed (and the credentialing application data will be visible on their data access application.)

That won't work anymore since proj.access_policy will never equal CREDENTIALED.  *Should* people be allowed to apply for data acess before being credentialed?

**Comment by tompollard** [comment]
@alistairewj Alistair, any thoughts on this? For self-managed projects, should users be allowed to submit an access request before becoming credentialed?

I don't see why the order in which people apply really matters, except that:

- changing behaviour to allow data requests before credentialing may require extra work (e.g. editing email templates).
- the person reviewing data requests may feel like they don't have information about non-credentialed users.

**Comment by alistairewj** [comment]
Yeah I think it's fair to only allow credentialed users to apply. As long as we can avoid the confusing dependencies, it'll make life a lot easier for the contributor review process.
